app/models/application_record.rb

```diff
@@ -2,4 +2,13 @@
 
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+  attribute :metadata, ActiveRecord::Type::Json.new
+
+  def metadata
+    { :user_capabilities => {} }
+  end
+
+  def user_context
```

Comment:
@hsong-rh This is going to be done for every instance, if you have 50 requests we will create this object 50 times each one making a direct call to RBAC slowing things do. We should try caching it one per session in the 
https://github.com/RedHatInsights/approval-api/blob/7248da7d6a8182cbc1ba73f651d8ff1035093c24/app/controllers/application_controller.rb#L22
With an ensure to clear out the cached value.